ui/static/ui/js/manage_taxonomies.jsx

```diff
@@ -185,7 +185,9 @@ define('manage_taxonomies', ['react', 'lodash', 'jquery', 'utils', 'bootstrap'],
         actionButtonClass: "btn btn-danger btn-ok",
         title: "Confirm Delete",
         message: "Are you sure you want to delete vocabulary '" +
-          this.props.vocabulary.name + "'?",
+          this.props.vocabulary.name + "'? ",
+        description: "All terms in the vocabulary will be deleted and" +
+          " removed from all learning resources in this repository.",
```

Comment:
Sorry, another change to the message based on a some internal focus-grouping:

```
"Deleting this vocabulary will remove it from all learning resources."
```

A nice side-effect: no more wrapping.